packages/quick-edit/functions/_middleware.ts

```diff
@@ -19,6 +19,8 @@ export const onRequest = async ({
 				"files.exclude": {
 					"*.d.ts": true,
 					"jsconfig.json": true,
+					"**/package.json": true,
+					"**/wrangler.toml": true,
```

Comment:
@penalosa We are hidding the `.d.ts` and `jsconfig.json` files on root only. Do we want to make it hidden on any directories as well? 

```suggestion
					"**/*.d.ts": true,
					"**/jsconfig.json": true,
					"**/package.json": true,
					"**/wrangler.toml": true,
```

Reply:
No—these settings are to catch files we inject at the root

Reply:
Although to be honest we also only need to catch wrangler.toml and package.json at the root, so maybe those don't need wildcards?

Reply:
Fixed on https://github.com/cloudflare/workers-sdk/pull/6546/commits/ff46d68cbf4d9f555b2b18350264c230cb15feab.